style.css

```diff
@@ -0,0 +1,1043 @@
+}
+
+.mem-list-main {
+  margin: 40px;
+  line-height: 35px;
+  font-size: 18px;
+  list-style: decimal;
+}
+
+.mem-list-main li::before {
+  content: "\2022";
+  color: #fff;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+  margin-left: -1em;
+}
+
+.mem-list-main a {
+  color: #fff;
+  text-decoration: underline;
+}
+
+.cat-firechat {
+  -o-object-fit: cover;
+  object-fit: cover;
+  display: block;
+  transform: rotate(-20deg);
+  transform-origin: left bottom;
+  box-shadow: rgba(0, 0, 0, 0.3) 0 2rem 4rem;
+  transition: all 0.2s ease-out 0s;
+  background-color: transparent;
+  position: relative;
+  z-index: 20;
+}
+
+.cat-firechat:hover {
+  transform: rotate(0deg);
+  box-shadow: rgba(0, 0, 0, 0.3) 0 0.5rem 2rem;
+  z-index: 10;
+}
+
+.membership-btn {
+  background-color: #333232;
+  margin: 0.75em;
+  font-size: 2rem;
+  color: #fff;
+  border-radius: 10px;
+  padding: 20px;
+  width: 308px;
+}
+
+.membership-btn:hover {
+  background-color: #727272;
+  transition: all 0.4s ease-out 0s;
+}
+
+.mem-img {
+  background-color: transparent;
+}
+
+/* Latest Posts Section */
+
+.latest-posts {
+  height: 600px;
+  display: flex;
+  margin-right: 45px;
+  margin-top: 20px;
+  margin-left: 10px;
+}
+
+.post-wrapper {
+  height: 550px;
+  position: relative;
+  z-index: 10;
+  margin-right: -50px;
+}
+
+.post-wrapper:hover {
+  transform: scale(1.05) translateY(-0.5px);
+  transition: all 0.5s;
+  z-index: 20;
+  background-color: rgb(184, 180, 180);
+}
+
+.post-intro {
+  font-size: 13px;
+  margin-left: 50px;
+  margin-right: 30px;
+  margin-top: 20px;
+  padding: 20px;
+}
+
+.post-arrow a {
+  color: white;
+  font-size: 3.5rem;
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.smashing-main a {
+  color: white;
+  text-decoration: underline;
+}
+
+.post-intro h1 {
+  margin-bottom: 20px;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.header-table-content a {
+  color: #fff;
+  text-decoration: underline #666;
+  font-size: 4rem;
+  text-align: center;
+  display: block;
+  line-height: 50px;
+  letter-spacing: 2px;
+  padding-bottom: 30px;
+  font-family: 'Ubuntu', sans-serif;
+}
+
+.footer-list a {
+  font-size: 24px;
+  color: white;
+  margin-left: 9px;
+}
+
+.statement a {
+  color: white;
+  text-decoration: underline;
+}
+
+.footer-links a {
+  margin-right: 15px;
+  text-decoration: underline;
+  color: white;
+}
+
+.guide a {
+  font-size: 20px;
+  padding: 14px;
+  font-family: sans-serif;
+  font-weight: 800;
+  border-radius: 10px;
+  background: rgb(87, 84, 84);
+}
+
+.post-intro h1 a {
+  text-decoration: none;
+}
+
+.post-intro h6 {
+  color: rgb(59, 53, 53);
+}
+
+.post-intro span {
+  color: rgb(59, 53, 53);
+}
+
+.post-intro h2 {
+  margin-top: 20px;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: 100;
+  line-height: 32px;
+}
+
+.post-author {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 15px;
+  font-weight: 800;
+}
```

Comment:
to make the code DRYer a good approach is to take out repeated properties and create their own selectors. In this case with the ```font-family```, my perception is that is used through all the file as the default font.

To improve it this could be done:

```
/* -- At the top of the file -- */
*,
* a {
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
}
```
or if the case is that only links and titles and subtitles are the ones that require this anther approach would be:
```
h1,
h2,
h3,
* a,
.post-intro,
.post-author {
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
}
```